<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1152</width>
      <height>864</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="MainPage">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(1, 34, 27);</string>
     </property>
     <widget class="QGroupBox" name="IncomeGB">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>50</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(19, 61, 51);
color: rgb(237, 255, 251);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="IncomeTitLbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>ПРИБЫЛЬ</string>
       </property>
      </widget>
      <widget class="QLabel" name="IncomeLbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 500 25pt &quot;Calibri&quot;;
color: rgb(0, 188, 0);</string>
       </property>
       <property name="text">
        <string>0 ₽</string>
       </property>
      </widget>
      <widget class="QPushButton" name="IncomeAddBtn">
       <property name="geometry">
        <rect>
         <x>287</x>
         <y>3</y>
         <width>30</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font: 700 12pt &quot;Calibri&quot;;
background-color: rgb(1, 34, 27);
border: none;
 }

QPushButton::hover{
	
	
	background-color: rgb(2, 70, 55);
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>106</y>
         <width>315</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/image/Graphic Up.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>label</zorder>
      <zorder>IncomeTitLbl</zorder>
      <zorder>IncomeLbl</zorder>
      <zorder>IncomeAddBtn</zorder>
     </widget>
     <widget class="QGroupBox" name="ExpenseGB">
      <property name="geometry">
       <rect>
        <x>415</x>
        <y>50</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(19, 61, 51);
color: rgb(237, 255, 251);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="ExpanceTitLbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>РАСХОДЫ</string>
       </property>
      </widget>
      <widget class="QLabel" name="ExpenceLbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 500 25pt &quot;Calibri&quot;;
color: rgb(202, 0, 0);</string>
       </property>
       <property name="text">
        <string>0 ₽</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ExpenceAddBrn">
       <property name="geometry">
        <rect>
         <x>287</x>
         <y>3</y>
         <width>30</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font: 700 12pt &quot;Calibri&quot;;
background-color: rgb(1, 34, 27);
border: none;
 }

QPushButton::hover{
	
	
	background-color: rgb(2, 70, 55);
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>107</y>
         <width>315</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/image/Graphic Down.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>label_2</zorder>
      <zorder>ExpanceTitLbl</zorder>
      <zorder>ExpenceLbl</zorder>
      <zorder>ExpenceAddBrn</zorder>
     </widget>
     <widget class="QGroupBox" name="BalanceGB">
      <property name="geometry">
       <rect>
        <x>775</x>
        <y>50</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(19, 61, 51);
color: rgb(237, 255, 251);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="BalanceTitLbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>БАЛАНС</string>
       </property>
      </widget>
      <widget class="QLabel" name="BalanceLbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 500 25pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>0 ₽</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BalanceAddBtn">
       <property name="geometry">
        <rect>
         <x>287</x>
         <y>3</y>
         <width>30</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font: 700 12pt &quot;Calibri&quot;;
background-color: rgb(1, 34, 27);
border: none;
 }

QPushButton::hover{
	
	
	background-color: rgb(2, 70, 55);
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Expence_DynamicGB">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>325</y>
        <width>1040</width>
        <height>240</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(19, 61, 51);
color: rgb(237, 255, 251);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="DynamicLbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>ДИНАМИКА РАСХОДОВ</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>90</width>
         <height>15</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl5">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>60</y>
         <width>90</width>
         <height>15</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl6">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl7">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>60</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl8">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>90</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl9">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>60</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl10">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>90</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl11">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>60</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="TypeExLbl12">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>90</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Вид расходов</string>
       </property>
      </widget>
      <widget class="QLabel" name="MonthLbl">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Месяц</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GoalGb">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>600</y>
        <width>501</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(19, 61, 51);
color: rgb(237, 255, 251);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="GoalLbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>ЦЕЛЬ</string>
       </property>
      </widget>
      <widget class="QLabel" name="GoalNowLBl">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>СЕЙЧАС</string>
       </property>
      </widget>
      <widget class="QLabel" name="GoalNeedLbl">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>60</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>ТРЕБУЕТСЯ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="GoalAddBtn">
       <property name="geometry">
        <rect>
         <x>468</x>
         <y>3</y>
         <width>30</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font: 700 12pt &quot;Calibri&quot;;
background-color: rgb(1, 34, 27);
border: none;
 }

QPushButton::hover{
	
	
	background-color: rgb(2, 70, 55);
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLabel" name="BalanceLbl_2">
       <property name="geometry">
        <rect>
         <x>117</x>
         <y>130</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 500 25pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>0 ₽</string>
       </property>
      </widget>
      <widget class="QLabel" name="BalanceLbl_3">
       <property name="geometry">
        <rect>
         <x>327</x>
         <y>130</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 500 25pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>0 ₽</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="InvestmentDB">
      <property name="geometry">
       <rect>
        <x>575</x>
        <y>600</y>
        <width>521</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(19, 61, 51);
color: rgb(237, 255, 251);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="InvestmentLbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>ИНВЕСТИЦИИ</string>
       </property>
      </widget>
      <widget class="QLabel" name="InvNALbl">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>105</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>НЕДОСТУПНО</string>
       </property>
      </widget>
     </widget>
     <zorder>IncomeGB</zorder>
     <zorder>BalanceGB</zorder>
     <zorder>Expence_DynamicGB</zorder>
     <zorder>GoalGb</zorder>
     <zorder>InvestmentDB</zorder>
     <zorder>ExpenseGB</zorder>
    </widget>
    <widget class="QWidget" name="IncomePage">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(1, 34, 27);</string>
     </property>
     <widget class="QGroupBox" name="IncomeAddGB">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>391</width>
        <height>701</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="NameIncomeLbl">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="SumIncomeLbl">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="CategoryIncomeLbl">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>190</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DateIncomeLbl">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="CancelIncomeBtn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>660</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SaveIncomeBtn">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>660</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="BackgroundLbl_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1152</width>
      <height>864</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(1, 34, 27);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>BackgroundLbl_3</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
